### PR TITLE
[av.eo.esperanto] Initial commit

### PR DESCRIPTION
**Pull request by anvalon**
Created the av.eo.esperanto.model lexical model.

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

**Comment by anvalon** [comment]
@LornaSIL @DavidLRowe I just tested the lexical model and didn't find anything else to change or fix. So I think the project is ready to be reviewed.

Thanks

File: release/av/av.eo.esperanto/av.eo.esperanto.kpj [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      <Filename>flag.png</Filename>
+      <Filepath>source\..\..\..\..\..\keyboards\release\e\eo_plus\source\flag.png</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.png</FileType>
+      <ParentFileID>id_be25819628403fa4f0a2f652eba09d93</ParentFileID>
+    </File>
+    <File>
+      <ID>id_0bf1020128a0de92f883ca3c3e61344b</ID>
+      <Filename>splash.png</Filename>
+      <Filepath>source\..\..\..\..\..\keyboards\release\e\eo_plus\source\splash.png</Filepath>
```
Comment (darcywong00) [inline release/av/av.eo.esperanto/av.eo.esperanto.kpj:93]: I think the paths to flag.png and splash.png can point to files you've attached on this PR.
e.g. source\flag.png and source\splash.png.

Our CI build doesn't have the keyboards repo available to reference files there.

Comment (darcywong00) [reply release/av/av.eo.esperanto/av.eo.esperanto.kpj:93]: These also aren't really needed in the .kpj file. But the paths in av.eo.esperanto.model.kps should be fixed.

**Comment by anvalon** [comment]
@darcywong00 Done, thanks.

**Review by darcywong00 — APPROVED** [review]
Thanks for the submission.  looks good to me!

**Comment by anvalon** [comment]
> Thanks for the submission. looks good to me!

Fine, thanks.

**Pull request merged**